libcontainer/container_linux.go

```diff
@@ -364,14 +364,8 @@ func (c *Container) start(process *Process) (retErr error) {
 func (c *Container) Signal(s os.Signal) error {
 	c.m.Lock()
 	defer c.m.Unlock()
-	status, err := c.currentStatus()
-	if err != nil {
-		return err
-	}
 	// To avoid a PID reuse attack, don't kill non-running container.
-	switch status {
-	case Running, Created, Paused:
-	default:
+	if !c.hasInit() {
```

Comment:
This seems also changed the logic about the container status?

Reply:
Do you have some problems when you using runc, feel free to open an issue.